src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Icons/IconsStyles.axaml

```diff
@@ -417,4 +432,23 @@
             <unifiedIcon:IconValue MdiValueSetter="mdi-alert-circle" />
         </Setter>
     </Style>
+
+    <Style Selector="unifiedIcon|UnifiedIcon.AlertOctagon">
+        <Setter Property="Value">
+            <unifiedIcon:IconValue MdiValueSetter="mdi-alert-octagon" />
+        </Setter>
+    </Style>
+
+    <Style Selector="unifiedIcon|UnifiedIcon.MonitorDiagnostics">
+        <Setter Property="Value">
+            <unifiedIcon:IconValue
+                GeometrySetter="M9.07495 19.25C8.80828 19.25 8.56662 19.175 8.34995 19.025C8.13328 18.875 7.98328 18.6834 7.89995 18.45L5.47495 12.75H1.19995V11.25H6.52495L9.07495 17.275L13.75 5.60005C13.8333 5.36672 13.9833 5.17505 14.2 5.02505C14.4166 4.87505 14.6583 4.80005 14.925 4.80005C15.1916 4.80005 15.4333 4.87505 15.65 5.02505C15.8666 5.17505 16.0166 5.36672 16.1 5.60005L18.525 11.25H22.8V12.75H17.475L14.925 6.77505L10.25 18.45C10.1666 18.6834 10.0166 18.875 9.79995 19.025C9.58328 19.175 9.34162 19.25 9.07495 19.25Z" />
```

Comment:
For these hardcoded geometries, you should add a short comment with the source. Something like "Made by the Design Team" just so we know where it's from.

Reply:
Yeah, good idea